pom.xml

```diff
@@ -37,10 +37,10 @@ THE SOFTWARE.
   <url>https://github.com/OctopusDeploy/octopus-jenkins-plugin/blob/master/readme.md</url>
 
   <scm>
-    <connection>scm:git:https://github.com/jenkinsci/octopusdeploy-plugin.git</connection>
-    <developerConnection>scm:git:https://git@github.com/jenkinsci/octopusdeploy-plugin.git</developerConnection>
-    <url>https://github.com/jenkinsci/octopusdeploy-plugin</url>
-    <tag>octopusdeploy-1.5.0</tag>
+    <connection>scm:git:https://github.com/OctopusDeploy/octopus-jenkins-plugin.git</connection>
```

Comment:
The `OctopusDeploy` repository is the root repository we will be using to deploy.